core/domain/collection_services_test.py

```diff
@@ -1357,19 +1365,19 @@ def test_paging(self):
         self.assertEqual(len(all_commits), 5)
         commit_dicts = [commit.to_dict() for commit in all_commits]
         self.assertDictContainsSubset(
-            self.COMMIT_ALBERT_CREATE_COLLECTION_2, commit_dicts[-1])
+            self.COMMIT_ALBERT_EDIT_COLLECTION_1, commit_dicts[-1])
         self.assertDictContainsSubset(
-            self.COMMIT_ALBERT_EDIT_COLLECTION_1, commit_dicts[-2])
+            self.COMMIT_ALBERT_EDIT_COLLECTION_2, commit_dicts[-2])
         self.assertDictContainsSubset(
-            self.COMMIT_ALBERT_EDIT_COLLECTION_2, commit_dicts[-3])
+            self.COMMIT_ALBERT_DELETE_COLLECTION_1, commit_dicts[-3])
         self.assertDictContainsSubset(
-            self.COMMIT_ALBERT_DELETE_COLLECTION_1, commit_dicts[-4])
```

Comment:
The reason why the code in the CommitLogUnitTests in both collection_services_test and exp_services_test had to change is that my new code to update first_published_msec when an exploration is first published adds increases the amount of commits in the original test code by 1. I had to slightly modify the test code to accommodate the extra commit.